services/github/github-created-at.tester.js

```diff
@@ -0,0 +1,9 @@
+import { createServiceTester } from '../tester.js'
+import { isFormattedDate } from '../test-validators.js'
+
+export const t = await createServiceTester()
+
+t.create('created at').get('/erayerdin/firereact.json').expectBadge({
+  label: 'created at',
+  message: isFormattedDate,
+})
```

Comment:
Could we add a test for the repo not found case please?

Reply:
Completed.